### PR TITLE
MPL-N to N-MPL

### PR DESCRIPTION
**Pull request by samcunliffe**
I couldn't resist trying (and failing) to get intersphinx to link up the objects. In any case, I've added two links to the project homepages directly for the first occurrences.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
 matplotlib-napari
```
Comment (dstansby) [inline docs/index.rst:1]:
```suggestion
napari-matplotlib
```

**Review by dstansby — CHANGES_REQUESTED** [review]

**Comment by dstansby** [comment]
Good catch!

File: docs/index.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-from one or more ``napari`` Layers.
+``napari-matplotlib`` is a `napari` plugin for generating `Matplotlib` plots
+from one or more `napari` Layers.
 
 Design
 ------
-``matplotlib-napari`` contains a number of different ``napari`` widgets. Each
-widget is designed to map one or more ``napari`` Layers on to a ``Matplotlib``
+``napari-matplotlib`` contains a number of different `napari` widgets. Each
+widget is designed to map one or more `napari` Layers on to a `Matplotlib`
```
Comment (samcunliffe) [inline docs/index.rst:9]:
```suggestion
widget is designed to map one or more `napari` Layers on to a `matplotlib`
```

**Review by dstansby — APPROVED** [review]

**Comment by dstansby** [comment]
:shipit:

**Pull request merged**